### ISSUE TITLE
When type is too long it gets cut

### ISSUE THREAD

**Issue by beuted** [GuillaumeNury/vscode-ts-auto-return-type#3]
When I use it on a function with a really long explicit type I get this:

![image](https://user-images.githubusercontent.com/2879233/60188209-3618f180-982f-11e9-905f-73d73f31a29a.png)

(This is not a visual thing this this actually the type stopping with "..."

**Comment by Zarepheth** [comment]
Having the same problem. Long inferred types are shortened with missing sections replaced by `...`.

Dealing with long, complex types is the reason I wanted this, but it is failing as badly as the VS Code tooltip (only difference I see between them is that the order of types is different).

**Comment by phiresky** [comment]
you can add

"noErrorTruncation": true, // https://github.com/microsoft/vscode/issues/64566#issuecomment-590140308

to the tsconfig (in your code as well as to fix this extension)

**Comment by GuillaumeNury** [comment]
Fixed in `1.1.0`. 🎉 

It is now working for:

```typescript
function veryLongReturnType(): { a0123456789: { b0123456789: { c0123456789: { d0123456789: { e0123456789: { a0123456789: { b0123456789: { c0123456789: { d0123456789: { e0123456789: number; }; }; }; }; }; }; }; }; }; b0123456789: { c0123456789: { d0123456789: { e0123456789: { a0123456789: { b0123456789: { c0123456789: { d0123456789: { e0123456789: number; }; }; }; }; }; }; }; }; c0123456789: { d0123456789: { e0123456789: { a0123456789: { b0123456789: { c0123456789: { d0123456789: { e0123456789: number; }; }; }; }; }; }; }; } {
  return {
    a0123456789: {
      b0123456789: {
        c0123456789: {
          d0123456789: {
            e0123456789: {
              a0123456789: {
                b0123456789: {
                  c0123456789: {
                    d0123456789: {
                      e0123456789: 12
                    }
                  }
                }
              }
            }
          }
        }
      }
    },
    b0123456789: {
      c0123456789: {
        d0123456789: {
          e0123456789: {
            a0123456789: {
              b0123456789: {
                c0123456789: {
                  d0123456789: {
                    e0123456789: 12
                  }
                }
              }
            }
          }
        }
      }
    },
    c0123456789: {
      d0123456789: {
        e0123456789: {
          a0123456789: {
            b0123456789: {
              c0123456789: {
                d0123456789: {
                  e0123456789: 12
                }
              }
            }
          }
        }
      }
    }
  }
}
```
